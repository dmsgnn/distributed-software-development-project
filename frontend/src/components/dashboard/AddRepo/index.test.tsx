import { axe } from 'jest-axe';

import { act, render } from 'test-utils';
import { AddRepo } from './';

describe('AddRepo', () => {
	it('should matches snapshot', () => {
		const { asFragment } = render(<AddRepo />);

		expect(asFragment()).toMatchSnapshot();
	});

	it('should be accessible', async () => {
		const { container } = render(<AddRepo />);

		let results: Awaited<ReturnType<typeof axe>>;
		await act(async () => {
			results = await axe(container);
		});

		expect(results).toHaveNoViolations();
	});
});
