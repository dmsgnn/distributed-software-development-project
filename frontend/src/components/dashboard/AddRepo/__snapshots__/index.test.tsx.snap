// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddRepo should matches snapshot 1`] = `
<DocumentFragment>
  <a
    class="chakra-button css-t68bru"
    href="/repos/new"
  >
    New Repo
  </a>
</DocumentFragment>
`;
